styling/media-queries/match-media/prefers-color-scheme/index.html

```diff
@@ -24,8 +52,36 @@
         map.appendChild(l);
       });
     </script>
-  <body>
-  <mapml-viewer projection="OSMTILE" zoom="9" lat="44.630550504861795" lon="-103.86611938476564" controls>
-  </mapml-viewer>
+
+    <mapml-viewer projection="OSMTILE" zoom="9" lat="44.630550504861795" lon="-103.86611938476564" controls>
+    </mapml-viewer>
+    <div class="description">
+      <h2>Introduction to Adaptive Color Scheme for <code>mapml-viewer</code></h2>
+      <p>This experiment demonstrates the integration of adaptive color schemes for the <code>mapml-viewer</code> custom element, enabling it to adjust between light and dark modes based on the user’s system preferences. By implementing a color scheme that responds to the OS or browser’s preferred settings, we aim to enhance visual comfort and accessibility across different environments.</p>
+      <p>The page automatically detects and applies the appropriate theme, allowing users to experience the map viewer seamlessly in both light and dark modes. This implementation uses the <code>&lt;meta name="color-scheme" content="dark light"&gt;</code> attribute to signal to the browser the compatibility with both modes, ensuring a responsive and user-friendly experience.</p>
+      <h3 class="desktop-only">How to Emulate OS Preferred Color Scheme in Browsers</h3>
+      <ul class="desktop-only">
+          <li><strong>Google Chrome:</strong>
+              <ul>
+                  <li>Go to <code>chrome://settings/appearance</code> and toggle the "Theme" to "System default" or "Dark."</li>
```

Comment:
@yushan-mu this doesn't actually work so I think we should take this advice out.  The only things that change the mode for the user are the devtools trick, or actually setting their mode to dark in the operating system.  As far as I know.